docs/getting-started.md

```diff
@@ -108,7 +109,7 @@ Tip: Je kan ook de [BAG Viewer](https://bagviewer.kadaster.nl/lvbag/bag-viewer/i
 ## Sluit aan op productie
 
 ### API key
-Neem contact op met de [BAG beheerder](https://zakelijk.kadaster.nl/bag) om een API key aan te vragen voor de productie omgeving van de BAG API. Voor de productieomgeving wordt dus een andere API-key gebruikt dan in de testomgeving.
+[Vraag een API key voor de productieomgeving aan](https://formulieren.kadaster.nl/aanvraag_bag_api_huidige_bevragingen_productie). Voor de productieomgeving wordt dus een andere API-key gebruikt dan in de testomgeving.
```

Comment:
Voor de productieomgeving gebruik je een andere API-key dan in de testomgeving. [Vraag een API key voor de productieomgeving aan](https://formulieren.kadaster.nl/aanvraag_bag_api_huidige_bevragingen_productie).